### PR TITLE
Add TEST_RUNNER for whitebox tests

### PR DESCRIPTION
**Pull request by bgurney-rh**
Related: #587

### CODE REVIEW

**Comment by bgurney-rh** [comment]
Ah, yes, isort.  I'll fix that.

File: src/stratis_cli/_errors.py [hunk 1]
```diff
@@ -399,6 +399,12 @@ class StratisCliEnvironmentError(StratisCliError):
     """
 
 
+class StratisCliTestRunError(StratisCliError):
```
Comment (mulkieran) [inline src/stratis_cli/_errors.py:402]: You'll want to define this class in the tests module not in the src. Probably the best place is in _misc.py, along with the definition of ```TEST_RUNNER```.

Comment (bgurney-rh) [reply src/stratis_cli/_errors.py:402]: On a simple move, I'm getting a few lint errors:

- undefined-variable (for StratisCliError; should I choose something else?)
- too-few-public-methods (0/2)

Comment (bgurney-rh) [reply src/stratis_cli/_errors.py:402]: I think I got it: change "StratisCliError" to "Exception".

Comment (bgurney-rh) [reply src/stratis_cli/_errors.py]: Resolved.

File: tests/whitebox/_misc.py [hunk 2]
```diff
@@ -171,6 +171,11 @@ def setUp(self):
 
 RUNNER = run()
 
+class StratisCliTestRunError(Exception):
```
Comment (mulkieran) [inline tests/whitebox/_misc.py:175]: Now make this a subtype of ```AssertionError```, that's the vital part that will the unittest framework to interpret the exception as indicating a test failure, not an error.

**Comment by bgurney-rh** [comment]
I'm closing this PR, as it is now being developed against the stratis-cli 2.1.0 branch.

**Pull request closed without merging**